modules/media/jsx/uploadForm.js

```diff
@@ -118,13 +118,14 @@ class MediaUploadForm extends React.Component {
               required={true}
               value={this.state.formData.visitLabel}
             />
-            <SelectElement
+            <DatalistElement
```

Comment:
I'm not too attached to this change but DatalistElement doesn't really mean much to me although I know that you are using a datalist element. What are your thoughts on maybe changing the name of the component to SearchElement?